QuizApp/Quiz/DataSource/QuizRepository.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+
+protocol QuizRepositoryProtocol {
```

Comment:
Repository is a separate layer, and it should therefore have its separate models e.g. **QuizRepoModel**
Since it is a separate layer, you should also make a group for it, as you did with other layers